ui/apps/platform/src/Containers/VulnMgmt/VulnMgmt.utils/getImageScanMessage.ts

```diff
@@ -13,6 +13,7 @@ export default function getImageScanMessage(
     const hasOSCvesUnavailable = scanNotes?.includes('OS_CVES_UNAVAILABLE');
     const hasOSCvesStale = scanNotes?.includes('OS_CVES_STALE');
     const hasCertifiedRHELCvesUnavailable = scanNotes?.includes('CERTIFIED_RHEL_SCAN_UNAVAILABLE');
+    const hasContentSetUnavailable = scanNotes?.includes('CONTENT_SET_UNAVAILABLE');
```

Comment:
The value of this variable is calculated, but then never used or exported.

Is it an artifact that needs to be deleted, or does it need to be wired up to something else?

Reply:
Good catch, I'll remove it. It's an artifact from when I was testing out how the FE part works.